src/funcTest/groovy/me/champeau/jmh/AbstractFuncSpec.groovy

```diff
@@ -25,19 +25,18 @@ import spock.lang.TempDir
 abstract class AbstractFuncSpec extends Specification {
 
     protected static final List<GradleVersion> TESTED_GRADLE_VERSIONS = [
-            GradleVersion.version('7.0'),
-            GradleVersion.version('8.0'),
+            GradleVersion.version('8.3'),
```

Comment:
If we still pin the old Shadow on 7.1.2, probably should keep the Gradle 7 in tests.

Reply:
I wanted to, but the new plugin requires Gradle >= 8.3.

Reply:
May have to add some version checks for it.

Reply:
true

(used my limited Groovy powers tho)